catalog/plugin/catalog-plugin-security-audit/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<blueprint xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+           xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+
+    <bean id="securityAuditPlugin"
+          class="org.codice.ddf.catalog.plugin.security.audit.SecurityAuditPlugin"
+          init-method="init"
+          destroy-method="destroy">
+        <cm:managed-properties
+                persistent-id="org.codice.ddf.catalog.plugin.security.audit.SecurityAuditPlugin"
+                update-strategy="container-managed"/>
+        <property name="auditAttributes">
+            <list>
+                <value>security.access-groups</value>
```

Comment:
Since all these Metacard attributes are defined in Alliance, we should be able to remove all these since they don't exist in DDF by default. @stustison @vinamartin Do you guys know if these should be default values in the blueprint/metatype? If this is the case, I'm curious if this feature should be off by default.